docs/components/marker.md

```diff
@@ -34,3 +34,15 @@ Offset of the marker from the left in pixels, negative number indicates left.
 
 ##### `offsetTop` {Number} - default: `0`
 Offset of the marker from the top in pixels, negative number indicates up.
+
+##### `captureScroll` {Boolean} - default: `false`
+Block map zoom when scrolling over this component.
+
+##### `captureDrag` {Boolean} - default: `true`
+Block map pan & rotate when dragging this component.
+
+##### `captureClick` {Boolean} - default: `true`
```

Comment:
I would elaborate here to say "capture mouse event at this component" or "prevent propagation of click to lower components, including map".